src/ngx_http_lua_kong_vars.c

```diff
@@ -53,12 +53,171 @@ ngx_http_lua_kong_variable_request_id(ngx_http_request_t *r,
 }
 
 
+static ngx_int_t
+ngx_http_lua_kong_get_ssl_raw_certificate(ngx_connection_t *c, ngx_pool_t *pool, ngx_str_t *s)
+{
+    size_t   len;
+    BIO     *bio;
+    X509    *cert;
+
+    s->len = 0;
+
+    cert = SSL_get_peer_certificate(c->ssl->connection);
```

Comment:
`SSL_get_peer_certificate` looks like a deprecated function. However I'm not sure what is the best practice here. I guess it should be fine to keep it as I also saw it being used elsewhere in nginx patches

Reply:
Good to know!